exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -45,7 +45,7 @@ drill.client: {
   supports-complex-types: true
 }
 
-drill.home: "/tmp"
+drill.dfs-home: "/tmp"
```

Comment:
On a standard HDFS setup, is there a /tmp folder? All the examples suggest that stuff often goes into "/user/something" such as "/user/drill".

Also, if "/tmp" behaves like a Linux /tmp, might something come along and clean up the UDFs during a long Drill run?